### PR TITLE
feat(YouTube) show toast on ReVanced error

### PR DESCRIPTION
**Pull request by LisoUseInAIKyrios**
Patch changes for https://github.com/revanced/revanced-integrations/pull/293

### CODE REVIEW

File: src/main/kotlin/app/revanced/patches/youtube/misc/debugging/patch/DebuggingPatch.kt [hunk 1]
```diff
@@ -38,11 +38,21 @@ class DebuggingPatch : ResourcePatch {
                         "revanced_debug_stacktrace_enabled",
                         StringResource(
                             "revanced_debug_stacktrace_enabled_title",
-                            "Print stack traces"
+                            "Debug log stack trace"
```
Comment (oSumAtrIX) [inline src/main/kotlin/app/revanced/patches/youtube/misc/debugging/patch/DebuggingPatch.kt:41]: Usually, we go with [Verb] [Noun] such as "Print stack traces", are you sure this is better?

**Review by oSumAtrIX — APPROVED** [review]

File: src/main/kotlin/app/revanced/patches/youtube/misc/debugging/patch/DebuggingPatch.kt (hunk 1 above)
Comment (LisoUseInAIKyrios) [reply src/main/kotlin/app/revanced/patches/youtube/misc/debugging/patch/DebuggingPatch.kt:41]: Maybe "debug logging stack trace"?

Comment (oSumAtrIX) [reply src/main/kotlin/app/revanced/patches/youtube/misc/debugging/patch/DebuggingPatch.kt:41]: If you want to indicate, that it applies ro logging, you can use "Log stack traces".
 out of the motivation that "Debuglogging" or "Debuglog" are not verbs.

**Review by oSumAtrIX — APPROVED** [review]

File: src/main/kotlin/app/revanced/patches/youtube/layout/sponsorblock/bytecode/patch/SponsorBlockBytecodePatch.kt [hunk 2]
```diff
@@ -263,7 +263,7 @@ class SponsorBlockBytecodePatch : BytecodePatch(
                         "replaceMeWithsetSponsorBarRect" ->
                             method.replaceStringInstruction(index, it, rectangleFieldName)
 
-                        "replaceMeWithsetMillisecondMethod" ->
+                        "replaceMeWithsetMillisecondMethod" -> // Edit: Where in integrations is this method used?  This might be dead code
```
Comment (oSumAtrIX) [inline src/main/kotlin/app/revanced/patches/youtube/layout/sponsorblock/bytecode/patch/SponsorBlockBytecodePatch.kt:266]: It seems like both are dead and can be removed.

**Pull request merged**